students/kulon_aleksander/lesson_03/exponentiation.py

```diff
@@ -0,0 +1,20 @@
+def power(a, n):
+    if n == 0:
+        return 1
+    if n > 1:
+        n = n - 1
+        a = a * power(a, n)
+        return a
+    else:
+        return a
+
+
+def main():
+    aa = float(input())
```

Comment:
When you moved this part to ```main()``` function you don't need to care about shadowing variables from outer scope - now the ```aa``` and ```nn``` variables are local variables used within the function. Of course now is ok :)